man/man8/zfs.8

```diff
@@ -441,6 +441,16 @@ For non-snapshots, the compression ratio achieved for the \fBused\fR space of th
 .sp
 .ne 2
 .na
+\fB\fBcreatetxg\fR
+.ad
+.sp .6
+.RS 4n
+The transaction group (TXG) in which the dataset or bookmark was created. It is suitable for ordering a list of snapshots, e.g. for incremental \fBsend\fR & \fBrecv\fR.
```

Comment:
I think it should be made more clear that a bookmark's `createtxg` is not the TXG in which the bookmark itself was created, but in which the snapshot from which the bookmark was derived from was created. Hence a snapshot and its bookmark (assuming both exist at the same time) will have the same `createtxg` value.